components/Settings/Settings.tsx

```diff
@@ -17,23 +22,41 @@ const _Settings = () => {
         onClick={() => setVisibility(!visible)}
       />
 
-      <Modal onCloseRequest={() => setVisibility(false)} visible={true}>
+      <Modal onCloseRequest={() => setVisibility(false)} visible={visible}>
         <div className="settings-content p-4">
-          <div className="settings-section-title">COLORS</div>
-
-          <div className="settings-row">
-            <span className="settings-row-label">Accidental Keys</span>
-            <Dropdown
-              label={() => (
-                <div
-                  className="rounded-sm cursor-pointer"
-                  style={{ height: 20, width: 20, backgroundColor: "green" }}
-                ></div>
-              )}
-            >
-              {() => <TwitterPicker />}
-            </Dropdown>
-          </div>
+          <div className="settings-section-title">Colors</div>
+          {[
+            { colorName: "naturalColor", label: "White Keys" },
+            { colorName: "accidentalColor", label: "Black Keys" }
+          ].map(({ label, colorName }) => (
+            <div key={colorName} className="settings-row">
+              <span className="settings-row-label">{label}</span>
+              <Dropdown
+                label={() => (
+                  <div
+                    className="rounded-sm cursor-pointer"
+                    style={{
+                      height: 20,
+                      width: 20,
+                      backgroundColor: theme[colorName]
+                    }}
+                  ></div>
+                )}
+              >
+                {() => (
+                  <TwitterPicker
```

Comment:
I actually much prefer having the hex codes there for power users. I think it's still user friendly for users who don't know color hex because they can just pick the color presets. I think it's a good balance between simplicity and flexibility.

Reply:
In that case, I will prefer a picker that gives equal opportunity to both type of users. I don't want that people who know hex should be able to use all the colors but others shouldn't. That's why I will prefer ChromePicker.

Reply:
Yes, I also like ChromePicker better! It's great! I updated.